src/style/variables.scss

```diff
@@ -0,0 +1,16 @@
+@import url('https://fonts.googleapis.com/css2?family=Quicksand:wght@400;600&display=swap');
+
+// Colors <---------
+
+$mainBackground: #C9D2Df;
+$lightBlueAccent: #E4E9EB;
+$lightTan: #F3E8E2;
+$salmon: #F4D2D0;
+$midBlue: #9EAEC5;
+$darkBlue: #7987A1;
+
+// Fonts <----------
+
+$primaryFont: 'Quicksand' 400, sans-serif;
```

Comment:
Sweet serifs are nice

Reply:
> Sweet serifs are nice

"Sans" is French for without. 😬

Reply:
lol hahaha looking at this again and that makes it make more sense! thanks @sciencefixion